lib/media/helper_modules/helpers.ex

```diff
@@ -565,24 +565,68 @@ defmodule Media.Helpers do
     end
   end
 
+  def convert_base64_to_file(base64_file) when is_binary(base64_file) do
+    case base64_file
+         |> Base.decode64() do
+      {:ok, file_binary} ->
+        %{size: _size, filename: filename} = S3Manager.get_base64_info(base64_file)
```

Comment:
@zaher-hs these functions to work with base64 files should belong to the helpers and not to the S3 Manager

Reply:
correct